tests/test_models/config/remote_service_model_raw_register.conf

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Thanks for the review. I considered whether the configs could be combined or not, but it was difficult to combine because ml-service type is determined by the initial setting value such as a sender or receiver.